.github/workflows/benchmark.yml

```diff
@@ -32,29 +33,31 @@ jobs:
           ruby-version: ${{ matrix.ruby }}
 
       - uses: actions/checkout@v4
+        with:
+          submodules: recursive
 
       - name: Install gems
         run: |
-          bundle config set --local without 'docs repl development'
+          bundle config set --local without 'development docs rbs repl'
           bundle install
 
-      - name: SDK Build
+      - name: Build SDK
         run: bundle exec rake build
 
-      - name: Benchmark
+      - name: Benchmark Gems
         env:
           JRUBY_OPTS: -J-Xmx4g
-        run: EXECUTION_ENV=github-action bundle exec rake benchmark
+        run: bundle exec rake benchmark:run
 
-      - name: configure aws credentials
+      - name: Configure AWS credentials
         uses: aws-actions/configure-aws-credentials@v4
         with:
-          role-to-assume: arn:aws:iam::469596866844:role/aws-sdk-ruby-performance-reporter
-          role-session-name: benchmark-report
-          aws-region: us-west-2
+          role-to-assume: arn:aws:iam::373952703873:role/BenchmarkReporter
+          role-session-name: benchmark-reporter
+          aws-region: us-east-1
```

Comment:
Curious to know why we are switching regions here?

Reply:
No particular reason other than our other team resources are in us-east-1 here.